data/README.md

```diff
@@ -0,0 +1,9 @@
+# Simorgh Data Samples
+
+These are a set of data samples which represent possbile input to Simorgh from upstream systems (and ultimately from a CMS).  They cover a number of scenarios, most of which are made up with random Lorem Ipsum or real text.
+
+Two of the scenarios, 25 and 27, are based on real articles published on the BBC News website.
```

Comment:
we should have separate sub-folders for articles e.g. /data/test and /data/prod

Reply:
with 25 & 27 in both?